Sources/ConverterScreen.swift

```diff
@@ -826,10 +826,10 @@ extension ConverterScreen: DiagramResultDelegate {
             cutShortNameFlag = self.getFlagToLabel(shortName: chosenCurrency!)
         }
         
-        if cutShortNameFlag == nil {
+        if chosenCurrency == nil {
```

Comment:
What about to refactor it with `if let chosenCurrency`?

Reply:
Corrected